modules/telnet/TelnetCommandManager.py

```diff
@@ -0,0 +1,377 @@
+from database.dbc.DbcDatabaseManager import DbcDatabaseManager
+# from database.realm.RealmDatabaseManager import RealmDatabaseManager
+from database.world.WorldDatabaseManager import WorldDatabaseManager
+from game.world.WorldSessionStateHandler import WorldSessionStateHandler
+from game.world.managers.objects.units.ChatManager import ChatManager
+from utils.TextUtils import GameTextFormatter
+from utils.constants.SpellCodes import SpellEffects, SpellTargetMask
+from game.world.managers.CommandManager import CommandManager
+
+from utils.Logger import Logger
+
+
+class TelnetCommandManager(CommandManager):
+    @staticmethod
+    def _handle_command(command_dict, parent_conn):
+        if parent_conn:
+            Logger.set_parent_conn(parent_conn)
```

Comment:
You are already doing this in `start_telnet` method, no? Maybe I'm mistaken but not sure if it's needed in both places.

Reply:
I believe I made that decision because I needed to assign it within each parallell process I wanted to access. I'm sending data two way between telnet processes and the other processes with multiprocessing.Pipe()

Reply:
I'll check if I could use some other method.